projects/fsastorefrontlib/src/cms-components/comparison-table/comaprison-table-panel-item/comparison-table-panel-item.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="product$ | async as product; else loading">
   <ng-container *ngIf="!isLoading; else loading">
-    <div class="table-header">
+    <div class="table-header sticky-position">
```

Comment:
Can BS's class `"sticky-top"` perhaps serve this purpose instead of custom new class?

Reply:
As we agreed I'll use `sticky-top`.